Mikado/tests/locus_test.py

```diff
@@ -2633,7 +2633,7 @@ def test_not_retained_neg(self):
             Abstractlocus._is_exon_retained((301, 1000),
                                             "-",
                                             segmenttree, graph, [(301, 470)],
-                                            introns=set.intersection(t1.introns, t2.introns),
+                                            introns=set.union(t1.introns, t2.introns),
```

Comment:
This is curious, how was this test passing before?

Reply:
The old version of the method always looked at the graph ... so using a missed intron would have been ignored.
This is actually better now